shared/actions/tracker2.js

```diff
@@ -0,0 +1,202 @@
+        result: 'error',
+      })
+    )
+
+const ignore = (_, action) =>
+  RPCTypes.identify3Identify3IgnoreUserRpcPromise({guiID: action.payload.guiID}, Constants.waitingKey)
+    .then(() =>
+      Tracker2Gen.createUpdateResult({
+        guiID: action.payload.guiID,
+        reason: `Successfully ignored`,
+        result: 'valid',
+      })
+    )
+    .catch(e =>
+      Tracker2Gen.createUpdateResult({
+        guiID: action.payload.guiID,
+        reason: `Failed to ignore`,
+        result: 'error',
+      })
+    )
+
+function* load(state, action) {
+  if (action.payload.fromDaemon) {
+    return
+  }
+  const guiID = state.tracker2.usernameToDetails.get(action.payload.assertion)
+  if (!guiID) {
+    throw new Error('No guid on profile 2 load? ' + action.payload.assertion || '')
+  }
+  yield* Saga.callRPCs(
+    RPCTypes.identify3Identify3RpcSaga({
+      incomingCallMap: {},
+      params: {
+        assertion: action.payload.assertion,
+        guiID: action.payload.guiID,
+        ignoreCache: !!action.payload.ignoreCache,
+      },
+      waitingKey: Constants.waitingKey,
+    })
+  )
+}
+
+// Just to bridge the old action
+const _getProfileOLD = (_, action) =>
+  Tracker2Gen.createLoad({
+    assertion: action.payload.username,
+    forceDisplay: action.payload.forceDisplay,
+    guiID: Constants.generateGUIID(),
+    ignoreCache: action.payload.ignoreCache,
+    inTracker: false,
+    reason: '',
+  })
+
+const loadFollow = (_, action) => {
+  const {assertion} = action.payload
+  const convert = fs =>
+    (fs.users || []).map(f => ({
+      following: f.isFollowee,
+      followsYou: f.isFollower,
+      fullname: f.fullName,
+      username: f.username,
+    }))
+  return (
+    !action.payload.inTracker &&
+    Promise.all([
+      RPCTypes.userListTrackers2RpcPromise({assertion, reverse: false}).then(convert),
+      RPCTypes.userListTrackers2RpcPromise({assertion, reverse: true}).then(convert),
+    ]).then(([followers, following]) => {
+      return Tracker2Gen.createUpdateFollowers({followers, following, username: action.payload.assertion})
+    })
+  )
+}
+
+function* tracker2Saga(): Saga.SagaGenerator<any, any> {
+  if (!flags.identify3) {
+    return
+  }
+
+  yield* Saga.chainAction<ConfigGen.SetupEngineListenersPayload>(
+    ConfigGen.setupEngineListeners,
+    setupEngineListeners
+  )
+
+  yield* Saga.chainAction<EngineGen.Keybase1Identify3UiIdentify3UpdateUserCardPayload>(
+    EngineGen.keybase1Identify3UiIdentify3UpdateUserCard,
+    updateUserCard
+  )
+  yield* Saga.chainAction<Tracker2Gen.ChangeFollowPayload>(Tracker2Gen.changeFollow, changeFollow)
+  yield* Saga.chainAction<Tracker2Gen.IgnorePayload>(Tracker2Gen.ignore, ignore)
+  yield* Saga.chainGenerator<Tracker2Gen.LoadPayload>(Tracker2Gen.load, load)
+  yield* Saga.chainAction<Tracker2Gen.LoadPayload>(Tracker2Gen.load, loadFollow)
+
+  // TEMP until actions/tracker is deprecated
```

Comment:
should this be before the `if (!flags.identify3)`?

Reply:
don't think so, tracker1 handles it itself